<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmFormas_pago</class>
 <widget class="QDialog" name="FrmFormas_pago">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Formas de Pago</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_36">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);
background-color: #304163;</string>
     </property>
     <property name="text">
      <string>Formas de Pago</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableView" name="tabla_buscar">
         <property name="styleSheet">
          <string notr="true">alternate-background-color: rgb(255, 255, 255);
font: 8pt &quot;Sans&quot;;
background-color: rgb(248, 245, 194);</string>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" rowspan="6">
           <widget class="QFrame" name="frame_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="topMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_borrar">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Borrar</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/Delete.png</normaloff>:/Icons/PNG/Delete.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btndeshacer_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Deshacer</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/undo.png</normaloff>:/Icons/PNG/undo.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGuardar_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Guardar</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/Save.png</normaloff>:/Icons/PNG/Save.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnEditar_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Editar</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/Save.png</normaloff>:/Icons/PNG/Save.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="shortcut">
                <string>Ctrl+S</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuscar_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Buscar</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/search.png</normaloff>:/Icons/PNG/search.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSiguiente_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Siguient&amp;e</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/Next.png</normaloff>:/Icons/PNG/Next.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAnterior_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>&amp;Anterior</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/Previous.png</normaloff>:/Icons/PNG/Previous.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAnadir_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>&amp;Nueva</string>
               </property>
               <property name="icon">
                <iconset resource="../maya.qrc">
                 <normaloff>:/Icons/PNG/add.png</normaloff>:/Icons/PNG/add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="4" rowspan="6">
           <widget class="QTextEdit" name="textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Dias de Pago:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; text-decoration: underline;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Introducir los días en los que se va a generar un vencimiento.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;RedFoX&lt;/span&gt; se encargará de calcular la fecha del proximo vencimiento considerando el día mayor más cercano.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Plazos:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; text-decoration: underline;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Días entre plazos: &lt;/span&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;RedFoX&lt;/span&gt; sumará estos días al ultimo vencimiento y buscará el día mayor más cercano.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Numero de Plazos:&lt;/span&gt; Valor mínimo 1, correspondiente a un solo vencimiento, incrementar hasta alcanzar el numero de vencimientos deseados. &lt;span style=&quot; text-decoration: underline;&quot;&gt;RedFoX&lt;/span&gt; se encargará de generar todos los vencimientos con la fecha correspondiente.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="rdbContado">
              <property name="text">
               <string>Pago al contado</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>Pago aplazado</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QGroupBox" name="groupBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Configuracion de pago aplazado</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0" colspan="8">
              <widget class="QLabel" name="label_14">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>17</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #304163;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Días de pago</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="7">
              <widget class="QSpinBox" name="spinDia_pago4"/>
             </item>
             <item row="1" column="6">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Dia pago 4:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinDia_pago1">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Dia pago 3:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Dia pago 2:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QSpinBox" name="spinDia_pago3"/>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="spinDia_pago2"/>
             </item>
             <item row="2" column="0" colspan="8">
              <widget class="QLabel" name="label_16">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>17</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #304163;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Plazos</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Dia pago 1:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Dias hasta el primer pago:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="dias_hasta_pago">
               <property name="maximum">
                <number>365</number>
               </property>
              </widget>
             </item>
             <item row="3" column="7">
              <widget class="QSpinBox" name="spinDias_entre_plazos">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="QSpinBox" name="spinNumero_plazos">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="3" column="2" colspan="2">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Numero de Plazos:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5" colspan="2">
              <widget class="QLabel" name="label_15">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Dias entre plazos:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="txtCuenta_contable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="txtcod_cuenta_contable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>F1 -Buscar</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" colspan="3">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Descripción</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="txtForma_pago"/>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="txtcod_forma_pago"/>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Codigo</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtcod_forma_pago</tabstop>
  <tabstop>btnAnadir_3</tabstop>
  <tabstop>btnAnterior_2</tabstop>
  <tabstop>btnSiguiente_2</tabstop>
  <tabstop>btnBuscar_2</tabstop>
  <tabstop>btnEditar_3</tabstop>
  <tabstop>btnGuardar_2</tabstop>
  <tabstop>btndeshacer_2</tabstop>
  <tabstop>btn_borrar</tabstop>
  <tabstop>tabla_buscar</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
